src/cljs/main/broadfcui/main.cljs

```diff
@@ -469,7 +481,8 @@
                      :auth2 (:auth2 @state)}])]]
       (footer)
       ;; As low as possible on the page so it will be the frontmost component when displayed.
-      [modal/Component {:ref "modal"}]])
+      [modal/Component {:ref "modal"}]
+      [tooltip/Container]])
```

Comment:
This whole "make sure the container's at the bottom of the page to avoid using z-index" thing, while reasonable, has always seemed awfully brittle to me.